docs/modules/policies/pages/compile.adoc

```diff
@@ -127,52 +129,55 @@ tests: <11>
-<15> Optionally set xref:configuration:engine.adoc#globals[globals] for this test. Applies to just this test and takes precedence over the global suite option.
-<16> Input to the policy engine
-<17> List of keys of principal fixtures to test
-<18> List of keys of resource fixtures to test
-<19> List of actions to test
-<20> Key of auxiliary data fixture to test (optional)
-<21> List of outcomes expected for each principal and resource. If a principal+resource pair specified in `input` is not listed in `expected`, then `EFFECT_DENY` is expected for all actions for that pair.
-<22> Key of the principal fixture under test. Use `principals` instead of `principal` if you want to specify identical expectations for multiple principals.
-<23> Key of the resource fixture under test. Use `resources` instead of `resource` if you want to specify identical expectations for multiple resources.
-<24> Expected outcomes for each action for each principal+resource pair. If an action specified in `input` is not listed, then `EFFECT_DENY` is expected for that action.
-<25> Optional list of xref:outputs.adoc[output values] to match
-<26> Name of the action that would produce the output
-<27> List of expected output values
-<28> List of keys of principal groups to test. You can provide this instead of, or as well as, `principals`.
-<29> List of keys of resource groups to test. You can provide this instead of, or as well as, `resources`.
-<30> Key of the principal group under test. You can provide this instead of, or as well as, `principal` or `principals`.
-<31> Key of the resource group under test. You can provide this instead of, or as well as, `resource` or `resources`.
+<4> Optionally set xref:configuration:engine.adoc#default_policy_version[default policy version] for this test suite
+<5> Optionally set xref:configuration:engine.adoc#lenient_scopes[lenient scope search] for this test suite
+<6> Optionally set xref:configuration:engine.adoc#globals[globals] for this test suite
+<7> Map of principal fixtures. The key is a string that can be used to refer to the associated principal.
+<8> Map of principal groups. The key is a string that can be used to refer to the associated group of principal fixtures.
+<9> Map of resource fixtures. The key is a string that can be used to refer to the associated resource.
+<10> Map of resource groups. The key is a string that can be used to refer to the associated group of resource fixtures.
+<11> Map of (optional) auxiliary data fixtures required to evaluate some requests. The key is a string that can be used to refer to the associated auxData.
+<12> List of tests in this suite
+<13> Name of the test
+<14> Optionally set options that apply to just this test. Test-specific options are not merged with suite-wide options, so any unspecified values revert to the default.
```

Comment:
I found [this behaviour](https://github.com/cerbos/cerbos/blob/018340b74e9d00eda655386c1f5ae54c205b265d/internal/verify/run_test_suite.go#L166-L169) slightly surprising, but given we can't distinguish between unset and zero values, I think we have to live with it - but it's worth documenting.